docs/interop/javascript.md

```diff
@@ -13,6 +13,9 @@ println(s"""libraryDependencies += "dev.zio" %%% "zio" % "${zio.BuildInfo.versio
 println(s"""```""")
 ```
 
+Java Time API which is not a part of ScalaJS. You might have to add dependecy that provides `java.time` package to
```

Comment:
Shall we point them to `scala-java-time` as one implementation that we know works in this regard?